src/credit_manager.c

```diff
@@ -62,24 +62,13 @@ struct credits_t *find_credit(char *name) {
   return NULL;
 }
 
-/*
- * Deposits credit for a user. Called in the credit manager (with input ==
- * NULL) or interactively when the user does not have enough money for his
- * intended purchase (with input == nickname).
- *
- */
```

Comment:
Why was the comment removed?

Reply:
Because this special case was removed

Reply:
I removed it because it documented the special case calling deposit_credit(NULL).